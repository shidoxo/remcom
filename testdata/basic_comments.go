package main

import "fmt"

// This is a package comment
func main() {
	// Simple line comment
	fmt.Println("Hello World") // End of line comment

	var x int // Variable declaration comment
	x = 42    // Assignment comment

	// Multiple consecutive comments
	// should be handled properly
	// and consolidated

	if x > 0 { // Inline condition comment
		fmt.Println("Positive") // Inside block comment
	}

	// Final comment at end
}
