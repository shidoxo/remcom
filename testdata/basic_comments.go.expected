package main

import "fmt"

func main() {
	fmt.Println("Hello World")

	var x int
	x = 42

	if x > 0 {
		fmt.Println("Positive")
	}

}
